src/EnergyPlus/HeatBalanceKivaManager.cc

```diff
@@ -1132,6 +1153,7 @@ Real64 KivaManager::getValue(
 	Real64 h = 0.0;
 	Real64 q = 0.0;
 	Real64 Tz = DataHeatBalFanSys::MAT( DataSurfaces::Surface( surfNum ).Zone ) + DataGlobals::KelvinConv;
+	assert(surfaceMap[surfNum].size() > 0);
```

Comment:
Is there an actual code path that allows this to happen?  Should it be protected by something more graceful like a severe/fatal?

Reply:
I cannot conceive of a way that this will this assert will be triggered in EnergyPlus. Perhaps in a future unit test implementation, though.